Example/Tests/ILGClasses_Tests_ObjC.m

```diff
@@ -35,8 +41,17 @@ - (void)testNonIncludedTestWorks
                                                    ]];
     
     NSSet *retrievedClasses = [ILGClasses classesPassingTest:^BOOL(Class class) {
-        NSString *className = NSStringFromClass(class);
-        return [className containsString:@"Class1"];
+        if ([ILGClasses class:class orAnyOfItsSuperclassesPassesTest:^BOOL(Class  _Nonnull class) {
+            return class == [NSObject class];
+        }]) {
```

Comment:
I'd prefer line-breaks before `class:` and `orAny…`, so that the whole block moved over, which I think would make it clearer what's inside the `if` condition and what's inside the "then" block.